Documentation/compatibility/wpf-PackageDigitalSignatureManager.DefaultHashAlgorithm-is-now-SHA256.md

```diff
@@ -0,0 +1,41 @@
+## The default hash algorithm for WPF PackageDigitalSignatureManager is now SHA256
+
+### Scope
+Edge Case
+
+### Version Introduced
+.NET 4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+The PackageDigitalSignatureManager provides functionality for digital signatures in relation to WPF packages.  Previously, the default algorithm (PackageDigitalSignatureManager.DefaultHashAlgorithm)
+used for signing parts of a Package was SHA1.  Due to recent security concerns with SHA1, this default has been updated to SHA256.  This change affects all package signing, including XPS documents.
```

Comment:
Package --> package
updated to SHA256 --> changed to SHA256 starting with the .NET Framework 4.7.1.